### PR TITLE
Add support for fudge dices.

### PR DESCRIPTION
**Pull request by eduardostalinho**
This PR adds support for [Fudge dice](https://en.wikipedia.org/wiki/Fudge_(role-playing_game_system)#Fudge_dice).

It can be rolled using `roll 4df`. Only subtract and add operations are supported.

### CODE REVIEW

**Comment by justinian** [comment]
Awesome, Fudge dice! I've been playing some FATE lately, and somehow didn't think to add this.
Thanks for the PR. I've just got a couple comments, but otherwise it looks good.

File: fudge.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		bonus, err := strconv.ParseInt(matches[2], 10, 0)
+		if err != nil {
+			return nil, err
+		}
+		result.Total += int(bonus)
+	}
+
+	for i := 0; i < len(result.Rolls); i++ {
+		roll := rand.Intn(3) - 1
+		result.Rolls[i] = roll
```
Comment (justinian) [inline fudge.go]: Why not add the rolls to the total here, so you don't have to loop again down below?

File: fudge_test.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+type fudgeSuite struct {
+}
+
+/*
+func (s *DiceSuite) SetUpSuite(c *C) {
+}
+
+func (s *DiceSuite) SetUpTest(c *C) {
+}
+*/
```
Comment (justinian) [inline fudge_test.go]: Since these are just commented out, it's probably better to just delete them.

File: fudge_test.go [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+var _ = Suite(&fudgeSuite{})
+
+/* =============================================================================
+ * Std dice test cases
```
Comment (justinian) [inline fudge_test.go]: Update this from `Std` to `Fudge`

**Review by justinian — CHANGES_REQUESTED** [review]

File: fudge.go (hunk 1 above)
Comment (eduardostalinho) [reply fudge.go]: You are right. This was just the first draft from StdDice.

I'm gonna change it.

**Comment by eduardostalinho** [comment]
updated @justinian

**Review by justinian — APPROVED** [review]

**Comment by justinian** [comment]
Awesome. Thanks a bunch, @eduardostalinho!

**Pull request merged**